server/src/domain/media/media.util.ts

```diff
@@ -564,7 +564,10 @@ export class VAAPIConfig extends BaseHWConfig {
 }
 
 export class RKMPPConfig extends BaseHWConfig {
-  getOptions(videoStream: VideoStreamInfo, audioStream?: AudioStreamInfo | undefined): TranscodeOptions {
+  getOptions(videoStream: VideoStreamInfo, audioStream?: AudioStreamInfo): TranscodeOptions {
+    if (this.config.twoPass) {
```

Comment:
`getOptions` calls `eligibleForTwoPass`, so configs that don't support two-pass override it to return false. You should do that here too for consistency.